frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -167,7 +170,20 @@ export const rerunPipelineAndStay: KebabAction = (kind: K8sKind, pipelineRun: Pi
 };
 
 export const rerunPipelineAndRedirect: KebabAction = (kind: K8sKind, pipelineRun: PipelineRun) => {
-  return rerunPipeline(kind, pipelineRun, null, { onComplete: handlePipelineRunSubmit });
+  return rerunPipeline(kind, pipelineRun, null, {
+    onComplete: handlePipelineRunSubmit,
+    label: 'Start Last Run',
+  });
+};
+
+export const rerunPipelineFromDetailPage: KebabAction = (
+  kind: K8sKind,
+  pipelineRun: PipelineRun,
+) => {
+  return rerunPipeline(kind, pipelineRun, null, {
+    onComplete: handlePipelineRunSubmit,
+    label: 'Rerun',
+  });
 };
```

Comment:
I think we need to move away from misusing `rerunPipeline` (RE: https://jira.coreos.com/browse/ODC-2184).

We do have a [rerunPipelineRun](https://github.com/openshift/console/pull/3606/files#diff-25f91b2db21c7ce21bcebedf9f18bb23R79), perhaps we do the same concept of [redirect](https://github.com/openshift/console/pull/3606/files#diff-25f91b2db21c7ce21bcebedf9f18bb23R172) vs stay... 

I've sparked a conversation in Slack in order to find out if we really want a non-redirect rerun from these screens.

Reply:
Okay, after a discussion with @vikram-raj - we won't do this change. The two methods are doing near the same thing and merging them now will just be slapping more duct tape on a growing problem. ODC-2184 will be responsible for taking a long, hard, look at making this file do what it was originally intended to do, give direct actions for Pipelines (and PipelineRuns).